src/providers/twitch/TwitchUser.hpp

```diff
@@ -3,13 +3,20 @@
 #include "util/RapidjsonHelpers.hpp"
 
 #include <pajlada/serialize.hpp>
+#include <QHashFunctions>
 #include <QString>
 #include <rapidjson/document.h>
 
 #include <cassert>
 
 namespace chatterino {
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+#    define QHashValue uint
+#else
+#    define QHashValue size_t
+#endif
```

Comment:
should be removable once the last comment has been resolved (switching out QSet for std::set/std::unordered_set)